java/api/pom.xml

```diff
@@ -14,11 +14,30 @@
   <description>java api for ray</description>
 
   <packaging>jar</packaging>
+  
+  <properties>
+    <jaxbVersion>2.3.0</jaxbVersion>
+  </properties>
 
   <dependencies>
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
+      <version>${jaxbVersion}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-core</artifactId>
+      <version>${jaxbVersion}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-impl</artifactId>
+      <version>${jaxbVersion}</version>
+    </dependency>
```

Comment:
Please define the full dependencies with versions in the [root pom file](https://github.com/ray-project/ray/blob/597abb24eaeeed7c9fd9a6142e975f64225f995b/java/pom.xml#L118), and only define groupId and artifactId here.

Reply:
done

Reply:
I was "forced" to keep the variable because it is the same value used in multiple dependencies (I might I've missed something but I tried it in the same way as for slf4j but maven complained...)